### PR TITLE
fix balancer views

### PR DESCRIPTION
**Pull request by mendesfabio**
I've checked that:

* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] the directory tree matches the pattern /sector/blockchain/ e.g. /tokens/ethereum
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each file has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`

### CODE REVIEW

File: ethereum/balancer/view_vebal_votes.sql [hunk 1]
```diff
@@ -76,7 +76,6 @@ WITH calendar AS (
         ON v.round_id <= r.round_id
         AND r.round_id < v.next_round
         AND r.end_timestamp <= v.unlocked_at
-        AND v.weight > 0
```
Comment (jeff-dude) [inline ethereum/balancer/view_vebal_votes.sql:79]: i'm unable to drop this view as 'dune_user_generated.vw_tianqi_gauge_reward' depends on it -- is this your user generated object? would you like to back it up, then drop, then allow this rebuild?

File: ethereum/balancer_v1/view_liquidity.sql [hunk 2]
```diff
@@ -58,7 +58,7 @@ CREATE MATERIALIZED VIEW balancer_v1.view_liquidity AS (
             1,
             2
         HAVING
-            SUM(sample_size) > 3
+            SUM(sample_size) > 5
```
Comment (jeff-dude) [inline ethereum/balancer_v1/view_liquidity.sql:61]: same comment here on this file with same user generated table restricting the rebuild

File: ethereum/balancer/view_vebal_votes.sql (hunk 1 above)
Comment (mendesfabio) [reply ethereum/balancer/view_vebal_votes.sql:79]: it's not mine but I know the user who maintains it. I don't think we need to back it up - I'm just fixing some issues in both views here and didn't change anything else.

**Review by jeff-dude — APPROVED** [review]
@mendesfabio updated

**Pull request merged**